slides/lesson12.md

```diff
@@ -37,35 +37,64 @@ let stuff = ["John", 20, true, undefined];
 
 ---
 
-### arrays and objects
+### accessing elements
 
-the difference between arrays and objects is that values in arrays don't need a name.
+we can access elements in the array by number
+
+the numbering starts at `0`:
 
 ```js
-// the value 25 has the name 'age'
-let person = {
-  age: 25
-};
-console.log(person.age); //25
+let cars = ["Saab", "Volvo", "BMW"];
 
-// how can I access the value 25 here?
-person = [25];
+console.log(cars[0]); // Saab
+console.log(cars[1]); // Volvo
+console.log(cars[2]); // BMW
```

Comment:
Looks great! What if in the last one instead of showing them `console.log(cars[2])` we ask them how they would retrieve the value 'BMW' from the array?